frontend/src/components/title.ts

```diff
@@ -0,0 +1,35 @@
+import { LitElement, type PropertyValues } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+function updateTitle(content: string) {
+  let title = document.head.querySelector<HTMLTitleElement>("title");
+
+  if (!title) {
+    title = document.createElement("title");
+    document.head.appendChild(title);
+  }
+
+  title.innerHTML = content;
```

Comment:
Setting `document.title` updates the `<title>` element as well!
```suggestion
  document.title = content;
```

Reply:
Oops didn't mean to push this commit, was debugging a test 😅